config/routes.rb

```diff
@@ -8,6 +8,7 @@
   resources :objects, only: %i[show] do
     member do
       get 'checksum'
+      get 'file', format: false
```

Comment:
i believe the header supplied mime type of the returned file will be irrelevant to the caller.

Reply:
Can you say more about adding `format: false` here?

Reply:
when the caller gets the file, are they truly likely to care about the mime type in the header?  I don't think we *allow* anything other than API calls to this URL, and the calling program should know how to parse what it gets, no?  Am I missing something?  I could remove this ...

Reply:
See https://guides.rubyonrails.org/routing.html#route-globbing-and-wildcard-segments, which I may be mis-understanding.

Reply:
@ndushay 
OK, so the net effect of having `format: false` here is raising a 404 if the router gets a request for `http://foo.bar/objects/druid:abc123/file.anyfileext` ? I don't see any harm in allowing the format param, since the controller action doesn't do anything with it, but I also don't see any harm in being more prescriptive about this route, so :+1: